profile/prune.go

```diff
@@ -22,6 +22,37 @@ import (
 	"strings"
 )
 
+var (
+	reservedNames = []string{"(anonymous namespace)", "operator()"}
+	bracketRx     = regexp.MustCompile("(" + strings.Replace(strings.Replace(
```

Comment:
Unfortunately, almost. QuoteMeta would also escape the "|" between reserved words, which we don't want here.